test_elasticsearch_async/test_connection.py

```diff
@@ -58,7 +58,7 @@ def test_error_is_properly_logged(connection, caplog, port, server):
 def test_timeout_is_properly_raised(connection, server):
     @asyncio.coroutine
     def slow_request():
-        yield from asyncio.sleep(0.01)
+        yield from asyncio.sleep(1)
```

Comment:
`read_timeout` is also a `TimeoutHandle`. From [client.py](https://github.com/aio-libs/aiohttp/blob/master/aiohttp/client.py#L206):

```
tm = TimeoutHandle(
    self._loop,
    timeout if timeout is not sentinel else self._read_timeout)
```

I'll look into this asap.